articles/20241126_Using_Environmental_Variables_in_Daytona.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Using Environmental Variables in Daytona'
+description: 'Learn how to set, manage, and use environmental variables in Daytona workspaces with a simple Python project for demonstration.'
+date: 2024-11-26
+author: 'Busayo Samuel'
+tags: ['Environment Variables', 'Development Environment', 'Daytona Workspaces']
+---
+
+# Using Environmental Variables in Daytona
+
+## Introduction
+
+[Environmental variables]('/definitions/20241126_definition_environment_variables.md') are important for configuring development environments and storing sensitive information. They provide a flexible way to control application behavior without hardcoding values directly into the source code. Daytona offers a streamlined method for setting, managing, and using environmental variables across various workspaces using the `daytona env` command. This simplifies configuration and enhances security especially when collaborating on projects.
+
+This article will explain using the `daytona env` command to set and manage environmental variables within Daytona workspaces. You will learn how to set variables, create a workspace, and access these variables within a workspace.
+
+### TL;DR
+
+- **What**: Learn to use `daytona env set` for managing workspace environmental variables
```

Comment:
@bellatrick please rewrite this following the examples on the blog, e.g. https://www.daytona.io/dotfiles/train-gpt-models-instantly-with-daytona-infrastructure